.github/workflows/test.yml

```diff
@@ -2,13 +2,12 @@
 name: Test
 
 on:
-  push:
-    branches:
-      - master
-    tags-ignore:
-      - 'debian/*'
   pull_request:
     types: [opened, reopened, synchronize]
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
```

Comment:
Should there be some steps after `workflow_dispatch`?

Reply:
I don't think so, based on the [docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) and [an example](https://github.com/linz/geostore/blob/bc04424d1675284b49273c7e4d6b7dd634a917ca/.github/workflows/cdk-bootstrap.yml#L3-L4).